.github/workflows/scripts/helpers/publish-packages-helper.mjs

```diff
@@ -0,0 +1,139 @@
+import { $ } from 'execa';
+import fs from 'fs';
+import { parseChangelog } from './helpers/changelog-parser.mjs';
+
+/**
+ * Gets the current version of the package from package.json
+ * @param {"@itwin/itwinui-react" | "@itwin/itwinui-variables"} pkg
+ * @returns {Promise<string>}
+ */
+const getCurrentPackageVersion = async (pkg) => {
+  const packageJson = JSON.parse(
+    fs.readFileSync(
+      `./packages/${pkg.substring('@itwin/'.length)}/package.json`,
+      'utf8',
+    ),
+  );
```

Comment:
node has JSON imports now if you want to use that.

```suggestion
const packages = new URL('../packages', import.meta.url).pathname;
const packageDir = pkg.substring('@itwin/'.length);

const { default: packageJson } = await import(
  `${packages}/${packageDir}/package.json`,
  { assert: { type: 'json' } }
);
```

Reply:
Ohh, I see. It works, but gives the following warning when using it with node 20:

```
(node:76706) ExperimentalWarning: Importing JSON modules is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
```

So, should we make this change?

Reply:
Hmm ok lets keep it as-is then.